Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue4720.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+using System.Threading;
+using System.Diagnostics;
+using System;
+using System.Threading.Tasks;
+using Xamarin.Forms.Controls.GalleryPages;
+
+#if UITEST
+using Xamarin.Forms.Core.UITests;
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.WebView)]
+#endif
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 4720, "UWP: Webview: Memory Leak in WebView", PlatformAffected.UWP)]
+	public class Issue4720 : TestNavigationPage
+	{
+		protected override void Init()
+		{
+			PushAsync(new Issue4720Content());
+		}
+
+#if UITEST
+		protected override bool Isolate => true;
+
+		[Test]
+		public async Task GitHub4720Test()
+		{
+			//4 iterations were enough to run out of memory before the fix.
+			int iterations = 10;
+
+			for (int n = 0; n < iterations; n++)
+			{
+				RunningApp.WaitForElement(q => q.Marked("New Page"));
+				RunningApp.Tap(q => q.Marked("New Page"));
+				RunningApp.WaitForElement(q => q.Marked("Close Page"));
+				await Task.Delay(2000);
+				RunningApp.Tap(q => q.Marked("Close Page"));
+			}
+		}
+
+#endif
+
+		[Preserve(AllMembers = true)]
+		public class _4720WebView : WebView
+		{
+		}
+
+		[Preserve(AllMembers = true)]
+		public class Issue4720WebPage : ContentPage
+		{
+			static int s_count;
+
+			public Issue4720WebPage()
+			{
+				Interlocked.Increment(ref s_count);
+				Debug.WriteLine($"++++++++ Issue4720WebPage : Constructor, count is {s_count}");
+
+				var label = new Label { Text = "Test case for GitHub issue #4720." };
+
+				var button = new Button { Text = "Close Page" };
+				button.Clicked += Button_Clicked;
+
+				var wv = new _4720WebView()
+				{
+					Source = new UrlWebViewSource { Url = "https://www.youtube.com/" },
```

Comment:
Can we change this to https://dotnet.microsoft.com/ ?

Reply:
It can be changed to any webpage, however to actually see the behavior reported by the issue creator, it seems that the webpage has to have a lot of media on it.  i don't think dotnet.microsoft.com would be "busy" enough for the test to fail without this enhancement.